core/src/main/scala/org/apache/spark/security/SocketAuthServer.scala

```diff
@@ -66,42 +87,45 @@ private[spark] abstract class SocketAuthServer[T](
 
 }
 
+/**
+ * Create a socket server class and run user function on the socket in a background thread.
+ * This is the same as calling SocketAuthServer.setupOneConnectionServer except it creates
```

Comment:
Seems we don't have setupOneConnectionServer anymore.

Reply:
Oops, good catch